PROJECT

```diff
@@ -94,4 +94,22 @@ resources:
   kind: Shovel
   path: github.com/rabbitmq/messaging-topology-operator/api/v1beta1
   version: v1beta1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: rabbitmq.com
+  group: rabbitmq.com
+  kind: SuperStream
+  path: github.com/rabbitmq/messaging-topology-operator/api/v1beta1
+  version: v1beta1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: rabbitmq.com
+  group: rabbitmq.com
+  kind: SuperStreamConsumer
+  path: github.com/rabbitmq/messaging-topology-operator/api/v1beta1
+  version: v1beta1
```

Comment:
I'd prefer to have the version as alpha, so that we don't tie ourselves into maintaining these CRDs for many releases afterwards. IIUC, some of the functionality here will be build into RabbitMQ Core and the intention is to fill an existing gap. It shounds to me as an alpha (experimental) state, more than a beta (pre-release stable).